packages/firebase_crashlytics/firebase_crashlytics/lib/src/utils.dart

```diff
@@ -20,13 +20,11 @@ List<Map<String, String>> getStackTraceElements(StackTrace stackTrace) {
         // Crashlytics Console groups issues with same stack trace.
         // Obfuscated stack traces contains abs address, virt address
         // and symbol name + offset. abs addresses are different across
-        // sessions, so same error can create different issues in Console.
-        // We replace abs address with '0' so that Crashlytics Console can
-        // group same exceptions. Also we don't need abs addresses for
-        // deobfuscating, if we have virt address or symbol name + offset.
-        final String method = frame.member.replaceFirstMapped(
-            _obfuscatedStackTraceLineRegExp,
-            (match) => '${match.group(1)}0${match.group(3)}');
+        // sessions, Crashlytics is smart enough to grouping those errors
+        // in the same issue. For iOS we use abs address for symbolication
+        // and for Android we use virt address.
+
+        final String method = frame.member;
```

Comment:
Can this whole block be simplified to not even check the regex, just do it for all `UnparsedFrame`s?

Something like:
```dart
if (frame is UnparsedFrame) {
  elements.add(<String, String>{
    'file': '',
    'line': '0',
    'method': frame.member,
  });
} else ...
```

Reply:
I see, I'll delete some redundant code and cleaned up the logic!